--- Shizuku-Rikka/manager/src/main/java/moe/shizuku/manager/home/HomeViewModel.kt	2026-02-12 18:00:34.818361036 +0000
+++ Shizuku-Pascua28/manager/src/main/java/moe/shizuku/manager/home/HomeViewModel.kt	2026-02-12 18:00:34.650361038 +0000
@@ -38,7 +38,8 @@
             }
         } else null
         val permissionTest =
-            Shizuku.checkRemotePermission("android.permission.GRANT_RUNTIME_PERMISSIONS") == PackageManager.PERMISSION_GRANTED
+            Shizuku.checkRemotePermission("android.permission.GRANT_RUNTIME_PERMISSIONS") ==
+                    PackageManager.PERMISSION_GRANTED || uid == 1000

         // Before a526d6bb, server will not exit on uninstall, manager installed later will get not permission
         // Run a random remote transaction here, report no permission as not running
